spec/generated/bundled-polaris-catalog-service.yaml

```diff
@@ -635,7 +635,7 @@ paths:
                 TableToUpdateDoesNotExist:
                   $ref: '#/components/examples/NoSuchTableError'
         '409':
-          description: Conflict - CommitFailedException, one or more requirements failed. The client may retry.
+          description: Conflict - Transaction commit failed due to concurrent modifications. The client may retry.
```

Comment:
This file is generated; the change should go into the appropriate spec and then this file needs to be regenerated.

Reply:
Good point about spec change.

However, the code fix in this PR does not require spec changes. I believe the client-facing behaviour is not changed by this PR.